sfr/subchannel/EBR-II/XX09_SCM_SS45R.i

```diff
@@ -0,0 +1,400 @@
+    boundary = inlet
+    area = S
+    mass_flux = ${mass_flux_in}
+    execute_on = 'timestep_begin'
+  []
+[]
+
+[Outputs]
+  exodus = true
+  csv = true
+[]
+
+[Postprocessors]
+  # [TTC-27]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 91
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.322
+  # []
+  # [TTC-28]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 50
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.322
+  # []
+  # [TTC-29]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 21
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.322
+  # []
+  # [TTC-30]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 4
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.322
+  # []
+  # [TTC-31]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 2
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.322
+  # []
+  # [TTC-32]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 16
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.322
+  # []
+  # [TTC-33]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 42
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.322
+  # []
+  # [TTC-34]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 80
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.322
+  # []
+  # [TTC-35]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 107
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.322
+  # []
+  # [MTC-20]
+  # type = SubChannelPointValue
+  # variable = T
+  # index = 33
+  # execute_on = 'TIMESTEP_END'
+  # height = 0.172
+  # []
+  # [MTC-22]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 3
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.172
+  # []
+  # [MTC-24]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 28
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.172
+  # []
+  # [MTC-25]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 60
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.172
+  # []
+  # [MTC-26]
+  #   type = SubChannelPointValue
+  #   variable = T
+  #   index = 106
+  #   execute_on = 'TIMESTEP_END'
+  #   height = 0.172
+  # []
+  [14TC-37]
+    type = SubChannelPointValue
+    variable = T
+    index = 52
+    execute_on = 'TIMESTEP_END'
+    height = 0.480
+  []
+  [14TC-39]
+    type = SubChannelPointValue
+    variable = T
+    index = 6
+    execute_on = 'TIMESTEP_END'
+    height = 0.480
+  []
+  [14TC-41]
+    type = SubChannelPointValue
+    variable = T
+    index = 40
+    execute_on = 'TIMESTEP_END'
+    height = 0.480
+  []
+  [14TC-43]
+    type = SubChannelPointValue
+    variable = T
+    index = 105
+    execute_on = 'TIMESTEP_END'
+    height = 0.480
+  []
+[]
+
+[Executioner]
+  type = Steady
+[]
+
+################################################################################
```

Comment:
you should make a separate file called XX09_output.i

and use !include XX09_output.i in both simulations